samples/OpenIdConnectSample/Program.cs

```diff
@@ -17,6 +17,7 @@ public static void Main(string[] args)
                     var serverCertificate = LoadCertificate();
                     options.UseHttps(serverCertificate);
                 })
+                .UseUrls("https://localhost:44318")
```

Comment:
Hardcode this URL so that the base address is consistent between `dotnet run` and VS F5.